source/renderer/app/components/news/NewsFeed.scss

```diff
@@ -95,6 +95,19 @@
     overflow: hidden;
     padding: 20px;
 
+    &.hasUpdate.emptyFeedContainer {
```

Comment:
@DeeJayElly can't we simply use `flex: 1` on `.newsFeedContainer` element?

Reply:
Fixed